contracts/facets/ArchaeologistFacet.sol

```diff
@@ -84,4 +86,77 @@ contract ArchaeologistFacet {
     {
         return s.cursedBonds[archaeologist];
     }
+
+    /// @notice Unwraps the sarcophagus.
+    /// @dev Verifies that the unencrypted shard matches the hashedShard stored
+    /// on chain and pays the archaeologist.
+    /// @param identifier The identifier of the sarcophagus
+    /// @param unencryptedShard The archaeologist's corresponding unencrypted shard
+    /// @return The boolean true if the operation was successful
+    function unwrapSarcophagus(
+        bytes32 identifier,
+        bytes memory unencryptedShard
+    ) external returns (bool) {
```

Comment:
You are correct it's probably not necessary. It may be we wanted the caller to have *some* value returned (which would always be true). Once we see how the caller catches errors (or reverts), we can remove this return value.